plugins/finder/jevents/jevents.php

```diff
@@ -384,19 +342,121 @@ protected function getItem($id)
 	 */
 	protected function getStateQuery()
 	{
-		$sql = $this->db->getQuery(true);
+		$db     = $this->db;
+		$query  = $db->getQuery(true);
+
```

Comment:
No - I'm just talking about changing $this->db->getQuery into 2 separate function calls